### PR TITLE
Reduce log verbosity

### PR DESCRIPTION
**Pull request by xcoulon**
see also https://github.com/codeready-toolchain/toolchain-common/pull/285

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

**Comment by mfrancisc** [comment]
Even here it would be nice if the logging library could provide a debug level (`logger.Debug(string)`) so that we can keep those logs turned on only when the operator is deployed in debugging mode for examlpe.

**Review by MatousJobanek — CHANGES_REQUESTED** [review]
TBH, I would love to keep these logs there because they are super helpful when analyzing logs for some troubleshooting.

**Comment by xcoulon** [comment]
> TBH, I would love to keep these logs there because they are super helpful when analyzing logs for some troubleshooting.

all of them?

File: go.mod [hunk 1]
```diff
@@ -31,6 +31,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.12.2
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20230111085908-1dc1cb70f1e9
```
Comment (mfrancisc) [inline go.mod:34]: friendly reminder

**Review by mfrancisc — APPROVED** [review]

**Review by MatousJobanek — APPROVED** [review]
Approving since it doesn't remove any logs now.
Thanks for changing it

**Pull request merged**